st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -97,6 +97,7 @@ def get_one(self, name, requester_user, scope=None, user=None, decrypt=False):
         key_ref = get_key_reference(scope=scope, name=name, user=user)
         extra = {"scope": scope, "name": name, "user": user, "key_ref": key_ref}
         LOG.debug("GET /v1/keys/%s", name, extra=extra)
+        LOG.audit("User %s decrypted the value %s ", user, name)
```

Comment:
Audit log messages support structured data via `extra` keyword argument so it would be good to include that here to make it easier to search.

For example:

```python
LOG.audit("User %s decrypted the value %s ", user, name, extra={"user": user, "key_name": name, "operation": "decrypt"})
```

Reply:
Thanks Kami! Ive updated it :)